latitude/index.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Latitude</title>
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    
+   
+      <main class="banner">
```

Comment:
1. check for the font properties, 
2. Footer is not matching,
3. check spacing for this page
![image](https://github.com/user-attachments/assets/a2f83fca-a046-4fa8-9d43-6df977f34c06)